src/test/java/picard/sam/AbstractAlignmentMergerTest.java

```diff
@@ -199,6 +200,136 @@
+                //            <sssssssMMMMMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMMiiMsssssss>
+                //                                         /\
+                //         with an insertion of two bases here should become
+                //       123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //            <ssssssSSSSSMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMSSSSsssssss>
+
+
+                {27, 14, 18, "7S20M", "17M2I1M7S", true, false, 18, 18,"11S16M","16M11S", false,
+                        default27LongR1Bases, default27LongR2Bases, default27LongR1Bases, default27LongR2Bases, null, null,
+                        default27LongR1Qualities, default27LongR2Qualities,default27LongR1Qualities, default27LongR2Qualities , null,null },
+
+
+                //       123456789.123456789.123456789.1234--56789.123456789.123456789.123456789.123456789
+                //            <sssssssMMMMMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMiiMMsssssss>
+                //                                        /\
+                //        with an insertion of two bases here should become
+                //       123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //            <ssssssSSSSSMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMSSSSsssssss>
+
+
+                {27, 14, 18, "7S20M", "16M2I2M7S", true, false, 18, 18, "11S16M", "16M11S", false,
+                        default27LongR1Bases, default27LongR2Bases, default27LongR1Bases, default27LongR2Bases, null, null,
+                        default27LongR1Qualities, default27LongR2Qualities, default27LongR1Qualities, default27LongR2Qualities, null, null},
+
+
+                //       123456789.123456789.123456789.1234--56789.123456789.123456789.123456789.123456789
+                //            <sssssssMMMMMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMiMMMsssssss>
+                //                                        /\
+                //        with an insertion of one bases here should become
+                //       123456789.123456789.123456789.123456789.123456789.123456789.123456789.123456789
+                //            <ssssssSSSSSMMMMMMMMMMMMMMMM
+                //                        MMMMMMMMMMMMMMMMSSSSsssssss>
+
+
+                {27, 14, 18, "7S20M", "16M1I3M7S", true, false, 18, 18, "11S16M", "16M11S", false,
+                        default27LongR1Bases, default27LongR2Bases, default27LongR1Bases, default27LongR2Bases, null, null,
+                        default27LongR1Qualities, default27LongR2Qualities, default27LongR1Qualities, default27LongR2Qualities, null, null},
+
+                //       123456789.123456789.123456789.1234--56789.123456789.123456789.123456789.123456789
```

Comment:
123456789.123456789.123456789.1234--56789.123456789.123456789.123456789.123456789
should be
123456789.123456789.123456789.1234-56789.123456789.123456789.123456789.123456789